docs/reference/setup/install/getting-started-tls-docker.asciidoc

```diff
@@ -0,0 +1,171 @@
+[[getting-started-tls-docker]]
+=== Getting started with TLS and Elasticsearch
+
+Starting with version 6.0.0, x-pack licensed for security (gold, platinum or enterprise subscriptions) https://www.elastic.co/guide/en/elasticsearch/reference/6.0/breaking-6.0.0-xes.html[requires SSL/TLS] encryption for the transport networking layer.
+
+This section demonstrates an easy path to get started with SSL/TLS for both http and transport using the elasticsearch-platinum docker image.
+
+For further details, please refer to {xpack-ref}/encrypting-communications.html[Encrypting Communications] and https://www.elastic.co/subscriptions[available subscriptions].
+
+==== Prepare the environment
+
+Inside a new, empty, directory create the following **four files**:
+
+`instances.yml`:
+["source","yaml"]
+instances:
+  - name: es01
+    dns:
+      - es01 <1>
+      - localhost
+    ip:
+      - 127.0.0.1
+  - name: es02
+    dns:
+      - es02
+      - localhost
+    ip:
+      - 127.0.0.1
+
+<1> Allow use of embedded Docker DNS server names.
+
+`.env`:
+[source,yaml]
+
+CERTS_DIR=/usr/share/elasticsearch/config/x-pack/certificates <1>
+ELASTIC_PASSWORD=PleaseChangeMe <2>
+
+<1> The path, inside the docker image, where certificates are expected to be found.
+<2> Initial password for the `elastic` user.
+
+[[getting-starter-tls-create-certs-composefile]]
+`create-certs.yml`:
+ifeval::["{release-state}"=="unreleased"]
+
+WARNING: Version {version} of Elasticsearch has not yet been released, so a
+`create-certs.yml` is not available for this version.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+["source","yaml",subs="attributes"]
+----
+version: '2.2'
+services:
+  create_certs:
+    container_name: create_certs
+    image: docker.elastic.co/elasticsearch/elasticsearch-platinum:{version}
+    command: >
+      bash -c '
+      if [[ ! -d config/x-pack/certificates/certs ]]; then
+      mkdir config/x-pack/certificates/certs;
+      fi;
+      if [[ ! -f /local/certs/bundle.zip ]]; then
+      bin/x-pack/certgen --silent --in config/x-pack/certificates/instances.yml --out config/x-pack/certificates/certs/bundle.zip;
+      unzip config/x-pack/certificates/certs/bundle.zip -d config/x-pack/certificates/certs; <1>
+      fi;
+      chgrp -R 0 config/x-pack/certificates/certs
+      '
+    user: ${UID:-1000}
```

Comment:
AsciiDoc is trying to expand this as an attribute and failing:
```
asciidoc: WARNING: getting-started-tls-docker.asciidoc: line 71: illegal system attribute name: UID
```

Standard escaping seems to work:
```
$\{UID:-1000\}
```

Reply:
Great catch! Addressed in d258de65303b51ed3e19539521fc8a69bff6badf